core/pom.xml

```diff
@@ -379,7 +385,7 @@
 		<dependency>
 			<groupId>com.github.mwiede</groupId>
 			<artifactId>jsch</artifactId>
-			<version>0.2.14</version>
+			<version>0.2.17</version>
```

Comment:
Dit vind ik redelijk eng maar goed.

Reply:
Enge dingen moeten we niet hebben 😉 Reverted.